src/Command/DaemonCommand.php

```diff
@@ -256,7 +256,7 @@ protected function loop(InputInterface $input, OutputInterface $output): void
             $this->dispatchEvent(DaemonLoopExceptionGeneralEvent::class);
 
             if ($this->getShowExceptions()) {
-                $this->getApplication()->renderException($e, $output);
```

Comment:
for me renderThrowable isnt available in sf4 ❓ 
so it is not retro compatible as the code seems to suggest.

Reply:
`renderThrowable` has been introduced in symfony 4.4

we could either drop support for symfony 4.3 => `"symfony/console" : "~4.4 || ~5.0",`

or still support symfony 4.3 => 
```
if (method_exists(renderThrowable)) { renderThrowable } 
else { renderException }
```

Reply:
Great catch!

Reply:
Changed to 4.4

Reply:
Dropped support for 4.3.